packages/components/src/box-control/axial-input-controls.js

```diff
@@ -113,27 +114,37 @@ export default function AxialInputControls( {
 			align="top"
 			className="component-box-control__vertical-horizontal-input-controls"
 		>
-			{ filteredSides.map( ( side ) => (
-				<UnitControl
-					{ ...props }
-					isFirst={ first === side }
-					isLast={ last === side }
-					isOnly={ only === side }
-					value={ side === 'vertical' ? values.top : values.left }
-					unit={
```

Comment:
For ease of review: the `unit` prop here was removed, and instead the parsing and potentially overriding of the unit is applied to the `value` prop